modules/tinymce/src/core/main/ts/dom/DomSerializerImpl.ts

```diff
@@ -19,6 +19,7 @@ import { isWsPreserveElement } from './ElementType';
 interface DomSerializerSettings extends DomParserSettings, WriterSettings, SchemaSettings, HtmlSerializerSettings {
   url_converter?: URLConverter;
   url_converter_scope?: {};
+  remove_trailing_brs?: boolean;
```

Comment:
Minor, keep the interface properties in alphabetical order.
```suggestion
  remove_trailing_brs?: boolean;
  url_converter?: URLConverter;
  url_converter_scope?: {};
```